packages/office-ui-fabric-react/src/components/__snapshots__/FocusZone.List.Example.tsx.shot

```diff
@@ -193,12 +193,6 @@ exports[`Component Examples renders FocusZone.List.Example.tsx correctly 1`] = `
             &:hover {
               opacity: 1;
             }
-            &.isHeader {
-              height: 32px;
-            }
-            .ms-Fabric--isFocusVisible & {
-              opacity: 1;
-            }
```

Comment:
Should this appear somewhere else in the snapshot output?

Reply:
Nah - it's not needed apparently.

Reply:
But in some other snapshots it is appearing...

Reply:
That's from getFocusStyles() that adds a `::after` thing. It is quite a different selector altogether... This one is not necessary (been testing it all day)